README.rst

```diff
@@ -111,8 +111,7 @@ For example, to access the raw text from the AG_NEWS dataset:
 
       >>> from torchtext.datasets import AG_NEWS
       >>> train_iter = AG_NEWS(split='train')
-      >>> next(train_iter)
```

Comment:
I think we can still do `next(iter(train))`. Maybe we can change it to that instead?

Reply:
This is true, but calling `iter(...)` first will make an exhaustible iterator, right? This could introduce misleading usages which result in issues like #1447. I may be wrong, though.